staging/src/k8s.io/api/apps/v1/types_swagger_doc_generated.go

```diff
@@ -96,7 +96,7 @@ func (DaemonSetSpec) SwaggerDoc() map[string]string {
 }
 
 var map_DaemonSetStatus = map[string]string{
-	"":                       "DaemonSetStatus represents the current status of a daemon set.",
+	"": "DaemonSetStatus represents the current status of a daemon set.",
```

Comment:
all of the whitespace changes are unexpected... what go version are you running with? try running `hack/update-gofmt.sh` before committing

Reply:
go1.13.7

Reply:
@liggitt I have properly run all the above mentioned scripts and committed the changes. Please review.